Rocket.Chat.iOS/ViewControllers/ViewController.swift

```diff
@@ -21,20 +22,30 @@ class ViewController: UIViewController, UITableViewDataSource, UITableViewDelega
     
     // indexPath to find the bottom of the tableview
     var bottomIndexPath:NSIndexPath = NSIndexPath()
-  
-    var messages: NSArray?
     
     var dateFormatter = NSDateFormatter()
     
-    
     var meteor: MeteorClient!
     
+    //JSON to keep the response
+    var chatMessages:JSON = []
+    
+    //Dictionary to keep the users from the JSON response
+    var users = [Int():[String(),String(),String(),String()]]
```

Comment:
Would it be possible to replace these dictionaries with an object?  It's really hard to understand what information will get stored here... ; )